components/Page.tsx

```diff
@@ -10,6 +10,7 @@ import SummaryContainer from "./SummaryContainer";
 import { DependencyData } from "../src/dataProcessing";
 import LoadingBackdrop from "./LoadingBackdrop";
 import HelpGuide from "./HelpComponents/HelpGuide";
+// import { CondensedHeaderSummaryContainer } from "./HeaderContainer";
```

Comment:
Is this import meant to be used here / is there a reason for the commented out implementation below (line 97)